build.gradle.kts

```diff
@@ -1,15 +1,21 @@
 buildscript {
     val kotlinVersion by System.getProperties()
     val dokkaVersion = "0.10.1"
+    val androidVersion = "4.0.1"
+    val composeVersion = "0.3.0-build134"
```

Comment:
I think we should centralize all dependencies versions in one place, currently, we could use `buildSrc` but `Gradle 6.9` will have [an official way to do manage them](https://docs.gradle.org/nightly/userguide/platforms.html#sub:central-declaration-of-dependencies).